app/javascript/src/utils/store.js

```diff
@@ -0,0 +1,8 @@
+export function debounced(cb, ms) {
```

Comment:
Let's rename `cb` to something else, I have no idea what it stands for

Reply:
Additionaly let's rename the file, it being under `utils/store` is a bit confusing to me. It can be it's own file, and with a more defined name `debouncedStore`, so it's clear this is only to be used for stores.

Reply:
`cb` stands for "callback", `ms` stands for `milliseconds`

Sure! How about
```suggestion
export function debounced(func, wait) {
```
to match the ones in `src/debounce.js`?

Reply:
As for the file name... maybe `storeUtils.js`? But it is already under `utils/`, so that is a bit redundant.

Reply:
Ah callback, that's a good name actually. Let's call it that but not abbreviated.

I don't mind if the file is singlepurpose and has a name to match. It doesn't need to contain more store utilities